compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -162,6 +162,12 @@ namespace jank::read
     {
     }
 
+    struct codepoint
+    {
+      char32_t character{};
+      size_t len{};
```

Comment:
We can use a `uint8_t` here, since we know that the character is no longer than 32 bits (`char32_t`).

Reply:
Highlighting this one again. All of our `size_t` usages for codepoints can be `uint8_t`. On a 64 bit machine, that's the difference between 1 byte and 8 bytes, which is significant.